bool            wait
int8            fps
Trajectory[]    trajectories
---
int8            error